helpers/addresses.py

```diff
@@ -458,6 +458,7 @@
         "frax": {
             "booster": "0xEe3Ab4e439ed52C7B8668864e2452ed23a9D67e8",
             "pool_registry": "0x41a5881c17185383e19Df6FA4EC158a6F4851A69",
+            "stk_badger_fraxbp": "0xb92e3fD365Fc5E038aa304Afe919FeE158359C88",
```

Comment:
i thought this was the stkcvx_ token at first, but apparently there is a difference between the stkcvx_ token received from the convex gauge and the wcvx_ token needed to deposit into the frax gauge.

probably better to name wcvx_ everywhere to make this diff explicit.

(so `Curve.fi Factory Crypto Pool: BADGER/FRAXBP Convex Deposit (0x25f0b7c3A7A43b409634a5759526560cC3313d75)` != `Staked Curve.fi Factory Crypto Pool: BADGER/FRAXBP Convex Deposit Frax (0xb92e3fD365Fc5E038aa304Afe919FeE158359C88)`)

Reply:
renamed it in 43a1a257525ee6663976203c8b279ac13d77fcfb